src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1018,7 +1017,7 @@ def list(cls, cmd, resource_group_name, environment_name):
 
 
 class SessionPoolPreviewClient():
-    api_version = SESSION_API_VERSION
+    api_version = PREVIEW_API_VERSION
```

Comment:
I will ask azure cli folks help to merge this PR: https://github.com/Azure/azure-cli-extensions/pull/8223

It will have some conflict with this PR.

Reply:
This PR has been merge. Please resolve the conflict.

Reply:
Merged and conflicts resolved.